### ISSUE TITLE
Customizable public key comment

### ISSUE THREAD

**Issue by rgov** [maxgoedjen/secretive#278]
Secretive includes a public key comment like `<key name>@secretive.<local hostname>`.

It would be nice to be able to override the comment so that it's easy to copy the key with my preferred comment.

Also as far as I know there's no harm in having spaces in the comment so it doesn't need to be formatted like a pseudo e-mail address.

**Comment by maxgoedjen** [comment]
Yeah, you're certainly not the first to want this 😅. The current version was sort of where I landed after the last request for this, but I'm not 100% happy with it tbh. Basically just trying to strike the right balance between customization and having a bunch of prefs. Let me think about this one a bit more.

**Comment by joachim-isaksson** [comment]
Couldn't one just use the secret name as comment in the details page too? SecretAgent gives "ecdsa-sha2-nistp256 <key> ecdsa-sha2-nistp256" as output of "ssh-add -L", but just changing a couple of lines;

- let curveData = writer.curveType(for: secret.algorithm, length: secret.keySize).data(using: .utf8)!
- keyData.append(writer.lengthAndData(of: curveData))
+ let label = secret.name.data(using: .utf8)!
+ keyData.append(contentsOf: writer.lengthAndData(of: label))

...gives the (according to me) much more friendly output "ecdsa-sha2-nistp256 <key> <key name>" which at least for me is more useful when agent-forwarding multiple keys and I can't remember which key is which. I assume the changes in the UI would also be fairly straight forward.

If this has been suggested before, please just ignore, I just started using Secretive so there may definitely be reasons for it working as it does that I don't know of, and I can live with building my own custom version :)

**Comment by agnosticlines** [comment]
+1, Would love this as a feature, not a fan of leaking my hostname and linking identities needlessly when using git ssh signing etc
